addons/toolbars/README.md

```diff
@@ -63,14 +72,25 @@ You should see a dropdown in your toolbar with options `light` and `dark`.
 
 Now, let's wire it up! We can consume our new `theme` global arg in a decorator using the `context.globalArgs.theme` value.
 
-For example, suppose you are using`styled-components`. You can add a theme provider decorator to your `.storybook/preview.js` config:
+For example, suppose you are using `styled-components`. You can add a theme provider decorator to your `.storybook/preview.js` config:
 
-```js
-const styledComponentsThemeDecorator = (storyFn, { globalArgs: { theme } }) => (
-  <ThemeProvider {...getTheme(theme)}>{storyFn()}</ThemeProvider>
-);
+```ts
+import { ThemeProvider } from 'styled-components';
+import { StoryContext, StoryGetter, StoryWrapper } from '@storybook/addons';
+
+const withThemeProvider: StoryWrapper = (Story: StoryGetter, context: StoryContext) => {
```

Comment:
ping @shilman: I am not certain what should be encouraged in the documentation: write decorators as functions or use the `makeDecorator()` api, so this was just a first guess here, I can update if needed

Reply:
This is perfect.  makeDecorator generally shouldn’t be used in user code